src/acl/Acl.h

```diff
@@ -62,11 +62,11 @@ class ACL
     /// \returns (linked) Options supported by this ACL
     virtual const Acl::Options &options() { return Acl::NoOptions(); }
 
-    /// configures ACL options, throwing on configuration errors
+    /// configures supported ACL options, throwing on configuration errors
```

Comment:
I would undo this change because the reader ought to assume that it is impossible to configure an unsupported option.
```suggestion
    /// configures ACL options, throwing on configuration errors
```